UI/src/app/components/comfy/comfy.component.html

```diff
@@ -78,8 +78,22 @@ <h4>Source tables
     </article>
     <article>
       <h4>Target tables
-        <ng-container *ngIf="data.target?.length">({{data.target.length}})</ng-container>
+        <mat-label style = "padding-right: 15px;" *ngIf="data.target?.length">({{data.target.length}})</mat-label>
```

Comment:
Please, don't use inline-styles. Use css class or add styles using tag instead of class and (write it in corresponding scss file)
Also don't use mat-label outside of formfields.
No need in mat-label there. Use simple html-tags